content/workshops/how-to-be-a-professional/_index.md

```diff
@@ -5,4 +5,7 @@ title: "How to be a professional"
 
 This is a run down of what it means to be a professional and how we expect you to behave within the tech department.
 
```

Comment:
please add `ready: true` to the frontmatter when you believe this is ready to be used